video/out/w32_common.c

```diff
@@ -1543,6 +1582,10 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
         if (w32->opts->fullscreen != w32->current_fs)
             reinit_window_state(w32);
         return VO_TRUE;
+    case VOCTRL_GET_FULLSCREEN:
+        *(int *)arg = w32->toggle_fs ^ w32->current_fs;
```

Comment:
You could simplify it as `w32->toggle_fs != w32->current_fs` (which is exactly the same because both fields are C99 bools).

I mean logically. toggle_fs is "whether the current fullscreen state needs to be switched", and current_fs is the real state. I guess my question is mostly about how toggle_fs is supposed to work.